### PR TITLE
[RFR] Move manual webmks console testcases

### PR DESCRIPTION
**Pull request by apagac**
__Moving__ manual testcases from `test_manual.py` to respective file.

I choose different approach this time. When testing consoles, there is considerable amount of combinations of operating systems, browsers and providers, all being tested manually at this time. Originally we had a testcase in polarion for every combination. Instead of copying all that from `test_manual.py`, I choose to describe the testing matrix in the comment of just one testcase.

Also, minor changes in testcase naming were performed.

Please let me know what you think.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f546cd25fe5fb424316bc1adb8dea2901e1bcdc5:

`cfme/tests/infrastructure/test_webmks_console.py`:
- :heavy_exclamation_mark: Line 258:15: **W291** *trailing whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/infrastructure/test_webmks_console.py [hunk 1]
```diff
@@ -174,3 +174,132 @@ def _validate_login():
...
+        - test_webmks_console_chrome_vsphere6_fedora
+        - test_webmks_console_edge_vsphere67_win10
+        - test_webmks_console_edge_vsphere65_win10
+        - test_webmks_console_edge_vsphere6_win10
+        - test_webmks_console_ie11_vsphere67_win2012
+        - test_webmks_console_ie11_vsphere67_win7
+        - test_webmks_console_ie11_vsphere65_win7
+        - test_webmks_console_ie11_vsphere65_win2012
+        - test_webmks_console_ie11_vsphere6_win2012
+        - test_webmks_console_ie11_vsphere6_win7
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_webmks_console.py]: @apagac overall, I like the approach but I have one concern. 
There are a lot of cases in one case, will it be easy to track which ones are passed already during execution?

Comment (apagac) [reply cfme/tests/infrastructure/test_webmks_console.py]: I know this has been merged, still want to get back to this question.

So all console tests are run manually due to huge number of combinations of browsers and systems. If some of the tests fail, I can always make a note in polarion when failing this whole testcase and put there that particular combination that failed.

In the future we might consider using `pytest_generate_tests` with a function that would take browser, provider and operating system and would generate the tests for us.

Comment (lina-nikiforova) [reply cfme/tests/infrastructure/test_webmks_console.py]: @apagac thanks! Sounds good :)

**Pull request merged**